### PR TITLE
Cross schema foreign keys cannot be procesed

### PR DESCRIPTION
**Pull request by ypyl**
[Issue](https://github.com/k1LoW/tbls/issues/358)

Not able to run the full set of tests within dev container (e.g. it is not clear how to run dynamodb related tests, also there are some fails with `test_ext_subcommand` from makefile and I really can't understand what it is doing and why it is required.

I will be happy to fix any issues and follow ups from you.
Thank you once again for the great utility.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (k1LoW) [inline .devcontainer/devcontainer.json:1]: I don't think this file needs to be committed.

**Comment by k1LoW** [comment]
@ypyl Thank you for your contribution!!!

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (RDBModel) [reply .devcontainer/devcontainer.json]: fixed

Comment (ypyl) [reply .devcontainer/devcontainer.json]: fixed in https://github.com/k1LoW/tbls/pull/383/commits/10e897eda3d5f9ff9abebcad264d899bbb6decd9

**Pull request merged**